README.md

```diff
@@ -7,6 +7,6 @@ Installation
 
     $ # go get github.com/cznic/goyacc
 
-    $ go get github.com/hsrx/goyacc
+    $ go get github.com/qsmx/goyacc
```

Comment:
I do not understand this change. Why should the package at `github.com/qsmx/goyacc` instruct people to install it using a command that actually installs a different package `github.com/qsmx/goyacc`? It does not make sense to me.

Please explain.